tests/test_copy.py

```diff
@@ -0,0 +1,97 @@
+import os
```

Comment:
This file is missing the Copyright header, as well as a docstring. 
You can run `make codestyle` in the root directory of kapitan and it will warn you about code style errors as well. :)

Reply:
Added the copyright, added docstring like the rest of the tests If I recall it correctly. :)
Ran `make codestyle` and fixed all style issues.